hiredis/version.py

```diff
@@ -1 +1 @@
-__version__ = "2.1.1"
+__version__ = "2.1.2"
```

Comment:
This will become 2.2.0 since it's not a minor change. Mind removing from this PR?

Reply:
Done.
Added an entry to the change log.